jx_sqlite/query_table.py

```diff
@@ -278,7 +278,7 @@ def query(self, query):
 
             output = Data(
                 meta={"format": "table"},
-                header=column_names,
+                header=[split_field(h)[0] if "\\" else h for h in column_names],
```

Comment:
What is split_field(h)[0] if "\\" else h` doing?  It appears to always be true; reducing to `split, _field(h)[0]`

Why are we taking the first part of the variable path?  The column names came from the "name" properties of edges, groupby and select, so they should be legitimate.